### PR TITLE
[Range] Adding samples to common Range methods

### PR DESCRIPTION
**Pull request by AlexJerabek**
These samples target commonly recorded actions that didn't already have sample code.

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excelscript/excelscript.rangefill.yml [hunk 1]
```diff
@@ -21,7 +42,25 @@ methods:
...
+
+          #### Examples
+
+          ```TypeScript
+          /**
+           * This script removes all fill color and styles from the used range.
+           */
+          function main(workbook: ExcelScript.Workbook)
+          {
+            // Get the used range of the current worksheet
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.rangefill.yml:56]:
```suggestion
            // Get the used range of the current worksheet.
```

**Review by alison-mk — APPROVED** [review]
Approving with two typo fixes.

File: docs/sample-scripts/excel-scripts.yaml [hunk 2]
```diff
@@ -1328,6 +1365,37 @@
...
       newSheet.activate();
     }
+'ExcelScript.RangeFill:interface':
+  - |-
+    /**
+     * This script sets the fill color of the used range to green.
+     */
+    function main(workbook: ExcelScript.Workbook)
+    {
+      // Get the used range of the current worksheet
```
Comment (alison-mk) [inline docs/sample-scripts/excel-scripts.yaml:1375]: @AlexJerabek Those missing periods are repeated in excel-scripts.yaml and snippets.yaml. Sorry I didn't mention this earlier!

Comment (AlexJerabek) [reply docs/sample-scripts/excel-scripts.yaml:1375]: I'll fix that in an upcoming PR before this gets merged to live.

**Pull request merged**